src/main/java/com/thealgorithms/datastructures/hashmap/hashing/MajorityElement.java

```diff
@@ -0,0 +1,26 @@
+import java.util.HashMap;
+
+public class MajorityElement {
+  
+    public static void majority(int[] nums){
+        HashMap<Integer,Integer> map = new HashMap<>();
+        int n = nums.length;
+        for (int i = 0; i < n; i++) {
+            if (map.containsKey(nums[i])){ //true
+                map.put(nums[i],map.get(nums[i])+1);
+            }else { //false or element appeared first time
+                map.put(nums[i],1);
+            }
+        }
+        for (int key: map.keySet()) {
+            if (map.get(key) > n/3){
+                System.out.println(key);
+            }
+        }
+    }
+  
+    public static void main(String[] args) {
```

Comment:
remove this main method and instead put some Junit tests. You could refer to https://github.com/TheAlgorithms/Java/tree/master/src/test/java/com/thealgorithms/datastructures/hashmap/hashing